pyproject.toml

```diff
@@ -73,6 +73,7 @@ dependencies = [
   "mypy==0.982",
   "psycopg2==2.9.5",
   "pytest==7.2.0",
+  "pytest-asyncio==0.20.2",
```

Comment:
I don't think this is needed, there's pytest anyio already used in other places. The same can be used here.

Reply:
indeed, i was wondering why your async tests were working and not mine, thanks for pointing out the secret! will change